src/item.cpp

```diff
@@ -1076,34 +1078,82 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
         it_tool* tool = dynamic_cast<it_tool*>(type);
 
         if ((tool->max_charges)!=0) {
-            std::string charges_line = _("Charges"); //;
+            int t_max;
+            const std::string t_ammo_name = _(ammo_name(tool->ammo_id).c_str());
```

Comment:
`ammo_name` should probably do the translation. I see at least 3 further calls to it where the result is not translated:

src/game.cpp:
```C++
add_msg(m_info, _("Out of %s!"), ammo_name(tool->ammo_id).c_str());
```

src/npctalk.cpp:
```C++
phrase.replace(fa, l, ammo_name(me->weapon.ammo_type()) );
```

src/player.cpp:
```C++
add_msg(m_info, _("That %s cannot be used on a %s."), used->tname().c_str(),
                       ammo_name(guntype->ammo).c_str());
```

That would also be kind of consistent with how `item::tname`, `monster::name`, etc. return a translated name.